invenio_cli/commands/upgrades.py

```diff
@@ -0,0 +1,54 @@
+# -*- coding: utf-8 -*-
+#
+# Copyright (C) 2020 CERN.
+#
+# Invenio-Cli is free software; you can redistribute it and/or modify it
+# under the terms of the MIT License; see LICENSE file for more details.
+
+"""Invenio module to ease the creation and management of applications."""
+
+from os import listdir
+
+from ..helpers.process import ProcessResponse, run_cmd, run_interactive
+from .steps import CommandStep
+
+
+class UpgradesCommands(object):
+    """Local installation commands."""
+
+    @staticmethod
+    def upgrade():
+        """Steps to install Python packages.
+
+        It is a class method since it does not require any configuration.
+        """
+        prefix = ['pipenv', 'run', 'invenio']
+        cmd1 = prefix + ['alembic', 'upgrade']
+        cmd2 = prefix + ['index', 'destroy', '--yes-i-know']
+        cmd3 = prefix + ['index', 'init']
+        cmd4 = prefix + ['rdm-records', 'rebuild-index']
```

Comment:
Could you put something more descriptive on the commands? I think that would also help you get a deep dive into what each of them does ;)

e.g.

```
        rebuild_idx_cmd = prefix + ['rdm-records', 'rebuild-index']
```

Reply:
I was trying to include few comments in order to avoid errors, but I will try to explain it.